charts/hub-agent/values.yaml

```diff
@@ -35,11 +35,11 @@ affinity: {}
 
 enableV1Alpha1APIs: false
 enableV1Beta1APIs: true
-enableClusterInventoryAPI: true
+enableClusterInventoryAPI: false
```

Comment:
Cause the latest release does not have this flag yet and i did not promote any pre-release because we have not tested in the RP side and not much confidence.